editor/components/post-saved-state/style.scss

```diff
@@ -1,3 +1,7 @@
+.editor-post-save-draft {
```

Comment:
Are these overriding styles? If so, wondering if we'd need added specificity in case the styles are applied out of the expected order.

Reply:
Pushed a fix to remove both this style, and the style it was (yes) overriding.

The code it overrode was this: https://github.com/WordPress/gutenberg/blob/master/components/icon-button/style.scss#L10 — the idea being that an icon button can have a label, but on mobile it should always just be the icon only. By applying a width and then removing that width past the mobile breakpoint, the labels would become visible as soon as there was screen realestate. Smarts that I thought were useful for the IconButton component.

But perhaps that was too smart. In the case of this PR, I wanted the Save label to be visible even on mobile, as the icon can't stand on its own, and the autosave feature isn't yet on par with other web-apps so you still need to press that button every once in a while. 

Also, removing the width doesn't seem to have had any negative effects as far as I can tell. So I removed it.